src/sentry/incidents/subscription_processor.py

```diff
@@ -293,6 +293,8 @@ def get_crash_rate_alert_metrics_aggregation_value(self, subscription_update):
         session_status = tag_key(self.subscription.project.organization.id, "session.status")
```

Comment:
`tag_key` could throw a `MetricIndexNotFound` exception here, which becomes more likely if we decide to manage string indexing on a per-organization basis. Since I don't know how far this exception would bubble up, I suggest we catch it here.

Reply:
I can create a follow up PR.. Although this is quite problematic .. when can this actually happen?

Reply:
Right now, it will not happen, because the string indexer stores one integer ID for every unique string, system-wide, and "session.status" was already added when the first session metric entered the system. But in the future we might decide to store an integer ID for every (org_id, unique string) tuple instead, and then this function throws if no session metric was ingested yet for this org.

Reply:
Yeah I understand what you mean now.. will create a follow up PR for this specifically